datalad_next/create_sibling_webdav.py

```diff
@@ -470,32 +543,29 @@ def _create_git_sibling(
         # e.g., it is not uncommon for credentials to be named after URLs
         remote_url += f'&dlacredential={urlquote(credential_name)}'
 
-    yield from ds.siblings(
-        # TODO set vs add, consider `existing`
-        action="add",
+    return ds.siblings(
+        action="add" if not reconfigure else "configure",
         name=name,
         url=remote_url,
-        # TODO probably needed when reconfiguring, but needs credential patch
-        fetch=False,
-        publish_depends=dependency,
-        return_type='generator',
+        publish_depends=publish_depends,
+        return_type='item-or-list',
```

Comment:
Unclear why this change to `return_type` is needed.

I would also like to keep `fetch=False` to indicate that this is the desired behavior.

Reply:
Done